frontend/docs/docs/develop/design-actions-dropdown.md

```diff
@@ -0,0 +1,21 @@
+# Actions Dropdowns
+
+While controls are always placed next to the most relevant content area, we expose most controls for an object in an _Actions_ dropdown menu to enable discovery of actions in a single place, and allow power-users to quickly accomplish tasks.
+
+## Implementation
+
+Actions dropdowns should generally contain a consistent set of actions for a given object. Whereas `---` symbolizes a horizontal separator, these actions should be ordered as follows:
+
+```txt
+Actions related only to the current page (eg: remove archived item from collection)
+---
+Actions related to editing the object (eg: edit workflow config, edit item metadata)
+Other actions related to the current object (eg: run crawl)
```

Comment:
Might also be worth including navigation to related items here, like we do for archived items where you can go to the related workflow from here

Reply:
The real answer here is they should be discouraged!  Will note that.

Reply:
Yeah? I could see it being useful sometimes. Maybe we have a consistent submenu for related items? `Go To Related > Workflow`

Reply:
Te considered a (in my opinion) small accessibility sin: https://www.makethingsaccessible.com/guides/site-navigation-is-not-an-aria-menu/